controllers/list/users/PKPUsersListHandler.inc.php

```diff
@@ -0,0 +1,378 @@
+				$contextDao = Application::getContextDAO();
+				$context = $contextDao->getByPath($this->_apiContextPath);
+				assert(is_a($context, 'Context'));
+			}
+			$this->_contextId = $context->getId();
+		}
+	}
+
+	/**
+	 * @copydoc ListHandler::getConfig()
+	 */
+	public function getConfig() {
+
+		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_DEFAULT);
+		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_GRID);
+		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_SUBMISSION);
+		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_USER);
+		AppLocale::requireComponents(LOCALE_COMPONENT_APP_DEFAULT);
+		AppLocale::requireComponents(LOCALE_COMPONENT_APP_USER);
+
+		$config = array();
+
+		if ($this->_lazyLoad) {
+			$config['lazyLoad'] = true;
+		} else {
+			$config['items'] = $this->getItems();
+			$config['itemsMax'] = $this->getItemsMax();
+		}
+
+		$config['apiPath'] = $this->_apiPath;
+		if (!empty($this->_apiContextPath)) {
+			$config['apiContextPath'] = $this->_apiContextPath;
+		}
+
+		$config['count'] = $this->_count;
+		$config['page'] = 1;
+		$config['getParams'] = $this->_getParams;
+
+		$config['isSiteAdmin'] = $this->_isSiteAdmin;
+		$config['mergeUserSourceId'] = $this->_mergeUserSourceId;
+
+		$config['filters'] = array(
+			'status' => array(
+				'heading' => __('common.status'),
+				'filters' => array(
+					array(
+						'param' => 'status',
+						'val' => 'active',
+						'title' => __('common.active'),
+					),
+					array(
+						'param' => 'status',
+						'val' => 'disabled',
+						'title' => __('common.disabled'),
+					),
+				),
+			),
+		);
+
+		if ($this->_contextId) {
+			$userGroupDao = DAORegistry::getDAO('UserGroupDAO');
+			$userGroupsResult = $userGroupDao->getByContextId($this->_contextId);
+			$userGroups = array();
+			while ($userGroup = $userGroupsResult->next()) {
+				$userGroups[] = array(
+					'param' => 'userGroupIds',
+					'val' => (int) $userGroup->getId(),
+					'title' => $userGroup->getLocalizedName(),
+				);
+			}
+			$config['filters']['userGroups'] = array(
+				'heading' => __('user.roles'),
+				'filters' => $userGroups,
+			);
+		} else {
+			$config['filters']['roles'] = array(
+				'heading' => __('user.roles'),
+				'filters' => array(
+					array(
+						'param' => 'userGroupIds',
+						'val' => 0,
+						'title' => __('user.roles.noneAssigned'),
+					),
+					array(
+						'param' => 'roleIds',
+						'val' => ROLE_ID_SITE_ADMIN,
+						'title' => __('default.groups.name.siteAdmin'),
+					),
+					array(
+						'param' => 'roleIds',
+						'val' => ROLE_ID_MANAGER,
+						'title' => __('default.groups.plural.manager'),
+					),
+					array(
+						'param' => 'roleIds',
+						'val' => ROLE_ID_SUB_EDITOR,
+						'title' => __('admin.roles.subeditors'),
+					),
+					array(
+						'param' => 'roleIds',
+						'val' => ROLE_ID_ASSISTANT,
+						'title' => __('user.role.journalAssistants'),
```

Comment:
When OMP comes we will then maybe need to rename the key?

Reply:
I've renamed the key for now. I started trying to shift the translations into pkp-lib, but it appears that a bunch of them use the term "Journal Assistants" or some other variant. So I've left the key in place in OJS and will remember to rename the key usage in OMP.

Reply:
I agree but how should we handle translations? I can't determine which ones use the word "journal" and which ones don't. Should I just remove all the existing translations here? 

When I was moving the translations over, I noticed a bunch of locales had existing translations for "Assistant" (singular), but that these often did not match what was in OJS for "Assistant". So it could be that in some translations there is a key difference being described. But I don't know...

Reply:
In this case I'm content to leave the translations as they are when we're not sure, and let OMP users correct them if needed.